.github/workflows/helm.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           python-version: 3.7
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.4.0
+        uses: helm/chart-testing-action@v2.3.1
```

Comment:
v2.4.0 is broken at the moment, see https://github.com/helm/chart-testing-action/issues/132
Downgrading fixes the issue. There's a fix on main but hasn't been released yet.

Reply:
Can you leave a comment for the downgrade with a reason? Something like:
```
# see https://github.com/helm/chart-testing-action/issues/132 before upgrading
uses: helm/chart-testing-action@v2.3.1
```

Reply:
not a blocker. :)

Reply:
Nice, thanks!